spec/mailers/product_mailer_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+RSpec.describe ProductMailer, type: :mailer do
+  describe '#share_email' do
+    let(:params) do
+      {
+        to_email: "example@platan.us",
+        product_name: "Chocolatera 2",
+        product_price: 2000.0,
+        product_link: "platan.us"
+      }
+    end
+    let(:mail) { described_class.with(params).share_email.deliver_now }
+    let(:from_email) { ENV.fetch('DEFAULT_EMAIL_ADDRESS') }
```

Comment:
En vez de depender de esa variable de ambiente, te tinca poner algo como esto? 

`allow(ENV).to receive(:[]).with(DEFAULT_EMAIL_ADDRESS).and_return('a@platan.us')` ?

Así puede ahorrarte también subir la variable de ambiente que te comenté más arriba

Reply:
Tome en consideración tu comentario, así que ahora evito llamar al ENV, y simplemente asigno el correo directamente.